AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -49,7 +49,6 @@ if (!caPasswordString.isNullOrBlank()) {
 
 repositories {
     mavenCentral()
-    mavenLocal()
```

Comment:
Issue: When working on my last PR to MPL,
I had issues getting the Test Vectors to run against HEAD.

Right now, isn't that .NET Test Vectors always runs against HEAD?
[Because it's project file refers to:](https://github.com/aws/aws-cryptographic-material-providers-library/blob/main/TestVectorsAwsCryptographicMaterialProviders/runtimes/net/TestVectors.csproj#L55)
```
<ProjectReference Include="../../../AwsCryptographicMaterialProviders/runtimes/net/MPL.csproj" />
```

But if you do not install MPL's head to Maven Local,
how can you get this same behavior in Java?

This difference was a significant annoyance to me,
as it was not documented,
and I did not realize it existed for the majority of Friday,
when I was trying to debug why .NET worked but Java did not
(Robin and I thought it was a bug in Dafny).

Reply:
Note: This is the wrong line for this comment, it should be on line 118, and line 89

Reply:
I talked to @seebees at Stand Up, there is a pattern we can use from the ESDK-Dafny that will fix this.

Reply:
I think we can resolve this conversation